DeezyMatch/tests/test_pipeline_ngram.py

```diff
@@ -8,13 +8,13 @@ def test_pipeline_ngram():
     from DeezyMatch import train as dm_train
     # train a new model
     dm_train(input_file_path="./inputs/input_dfm_pytest_002.yaml",
-             dataset_path="./dataset/dataset-string-similarity_test.txt",
+             dataset_path="./dataset/dataset-string-matching_train.txt",
              model_name="test002")
     
     from DeezyMatch import finetune as dm_finetune
     # fine-tune a pretrained model stored at pretrained_model_path and pretrained_vocab_path
     dm_finetune(input_file_path="./inputs/input_dfm_pytest_002.yaml",
-                dataset_path="./dataset/dataset-string-similarity_test.txt",
+                dataset_path="./dataset/dataset-string-matching_train.txt",
```

Comment:
@mcollardanuy finetune?

Reply:
aaaa, I see. Are you using only matching_train in this for testing?

Reply:
Ah, yes, I just used the same dataset throughout in this test just to make sure the last test does not fail (because candidates and queries are the same). But we can definitely use `finetune` and `test` in the relevant functions.

Reply:
I think it is good to have this example/test as well.